radix-engine/src/engine/process.rs

```diff
@@ -832,7 +692,10 @@ impl<'r, 'l, L: SubstateStore> Process<'r, 'l, L> {
         let moving_proofs = self.send_proofs(&output.proof_ids, MoveMethod::AsReturn)?;
 
         // drop proofs and check resource leak
-        self.drop_all_proofs()?;
+        self.drop_all_named_proofs()?;
+        if let Some(auth_zone) = &mut self.auth_zone {
+            auth_zone.clear();
+        }
```

Comment:
Can we have a `clear_all` method on the auth zone?

Reply:
You mean rename?

Reply:
I mean can we SInvoke the `clear_auth` method of the auth zone?

Reply:
This is fixed in a later PR so will just merge as is.